e3sm_to_cmip/resources/default_handler_info.yaml

```diff
@@ -277,19 +277,14 @@
   positive: up
   unit_conversion: g-to-kg
 
-- cmip_name: sftlf
```

Comment:
Add these back in. You dont need to use them if you dont want, but they're used in the E3SM model and shouldnt be removed unless there's some reason.